test/functional/terminal/tui_spec.lua

```diff
@@ -280,36 +280,30 @@ describe('tui focus event handling', function()
   end)
 
   it('can handle focus events in terminal mode', function()
-    feed_data(':set shell='..nvim_dir..'/shell-test\n')
-    feed_data(':set noshowmode laststatus=0\n')
-
-    retry(2, 3 * screen.timeout, function()
```

Comment:
Hm, I haven't seen the test failing locally. The test previously used the `shell-test` utility, which immediately exits. It now uses `tty-test`, to emulate a running shell. I'd suggest to keep it this way and revert to the retrying version if spurious failures occur in the future.